pkg/cloudprovider/amifamily/resolver.go

```diff
@@ -44,7 +44,7 @@ var DefaultEBS = v1alpha1.BlockDevice{
 
 // Resolver is able to fill-in dynamic launch template parameters
 type Resolver struct {
-	amiProvider      *AMIProvider
+	AmiProvider      *AMIProvider
```

Comment:
I had written a function in resolver.go so that we dont need to expose it and the launch template provider was calling that function. But since the args of the function were same as `r.amiProvider.Get`, as a part of this [conv](https://github.com/aws/karpenter/pull/2812#discussion_r1024415985) I removed that function and exposed this amiProvider.